src/pages/chaos/index.tsx

```
@@ -0,0 +1,141 @@
+              return <tr>
+              <td align='left'>
+                <Typography size='xs' className='pt-4' secondary>{index+1}.</Typography>
+              </td>
+              <td align='center'>
+              <Typography size='md' className='pt-4' truncate >{address}</Typography>
+              </td>
+              <td align='right'>
+              <Typography size='md' className='pt-4' >{thousandSeperator(totalChaosRewards)}</Typography>
+              </td>
+            </tr>
+            })}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)